packages/superset-ui-demo/storybook/shared/components/ResizableChartDemo.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from '@superset-ui/style';
+import React, { useState, ReactNode } from 'react';
+import { DecoratorFunction } from '@storybook/addons';
+import ResizablePanel, { Size } from './ResizablePanel';
+
+export const SupersetBody = styled.div`
+  background: #f5f5f5;
+  padding: 16px;
+  min-height: 100%;
+
+  .panel {
+    margin-bottom: 0;
+  }
+`;
+
+export default function ResizableChartDemo({
+  children,
+  panelPadding = 30,
```

Comment:
`padding` for cells in Superset dashboard is `16px`. should this be `32`?
I think `padding` usually means one side. May be better make this padding 16 and do the `*2` when computing children width/height.